custom/icds_reports/ucr/data_sources/dashboard/birth_preparedness_forms.json

```diff
@@ -251,6 +443,24 @@
           "type": "property_path",
           "property_path": ["form", "meta", "userID"]
         }
+      },
+      "anc_details": {
+        "type": "nested",
+        "argument_expression": {
+          "type": "array_index",
+          "array_expression": {
+	        "type": "property_path",
+	        "property_path": ["form", "bp1", "iteration", "item"]
+          },
+          "index_expression": {
+            "type": "constant",
+            "constant": 0
```

Comment:
Is it intentional that this will always take the first item here? Should it coalesce all values?

Reply:
Yeah, there can only be one anc visit (and therefore anc_details section) per bp form, confirmed that with the app team as well